sdk/github.md

```diff
@@ -205,8 +205,28 @@ tied in with the [SDK Upload Action](https://github.com/ably/sdk-upload-action).
 
 ## Labels
 
-We are slowly converging on a set of labels that are useful for our context and these are being duplicated across client libraries when needed.
-This list is canonically defined [here](https://github.com/ably/ably-common/blob/main/github/labels.yml).
+The following table canonically defines labels we use in common across our open source SDK repositories:
+
+| Name | Color | Description | Additional Notes |
+| ---- | ----------- | ----- | ---------------- |
+| `blocked-by-ably` | ![fbca04](https://img.shields.io/badge/-fbca04-fbca04) | We can't proceed until something under our direct control, in a different codebase, happens. | Comments should be added to indicate what the blockage is (e.g. another SDK repository). |
+| `blocked-by-external` | ![fbca04](https://img.shields.io/badge/-fbca04-fbca04) | We can't proceed until something outside of Ably's direct control happens. | Comments should be added to indicate what the blockage is. |
+| `bug` | ![d73a4a](https://img.shields.io/badge/-d73a4a-d73a4a) | Something isn't working. It's clear that this does need to be fixed. | Usually implies that related changes can be released in a `patch` version bump. |
+| `breaking` | ![ec4b42](https://img.shields.io/badge/-ec4b42-ec4b42) | Affects the developer experience when working in our codebase. | Implies a need to release related changes in a `major` version bump. |
+| `code-quality` | ![ff88cc](https://img.shields.io/badge/-ff88cc-ff88cc) | Affects the developer experience when working in our codebase. |
```

Comment:
I am not sure whether this label would only be applicable for the cases where user experience is affected. Shouldn't any internal code quality improvements (that are invisiible to our users) be part of this?

Reply:
That's what was intended to be meant by "in our codebase". Could you suggest better wording that makes it clearer?

Reply:
Does this help?: https://github.com/ably/engineering/pull/44/commits/a2f1ce07c6b840d2902d45e70ee1ea71d09ea174

Reply:
Ah of course. I think my mind quickly jumped to the user as we use 'developer experience' on the context of our SDK users 🤦